tool/actions-plan-preview/main.go

```diff
@@ -141,8 +141,33 @@ func main() {
 			doComment(failureBadgeURL + "\nUnable to run plan-preview for a closed pull request.")
 			return
 		}
+
+		// Find comments we sent before
+		comment, err := findLatestPlanPreviewComment(ctx, ghGraphQLClient, event.Owner, event.Repo, event.PRNumber)
+		if err != nil {
+			log.Printf("Unable to find the previous comment to minimize (%v)\n", err)
+		}
+
 		body := makeCommentBody(event, result)
-		doComment(failureBadgeURL + "\n" + body)
+		doComment(body)
+
+		if comment == nil {
+			log.Println("plan-preview result has error")
+			os.Exit(1)
+			return
+		}
+
+		if bool(comment.IsMinimized) {
+			log.Printf("Previous plan-preview comment has already minimized. So don't minimize anything\n")
+			return
+		}
+
+		if err := minimizeComment(ctx, ghGraphQLClient, comment.ID, "OUTDATED"); err != nil {
+			log.Printf("warning: cannot minimize comment: %s\n", err.Error())
+			return
+		}
```

Comment:
[IMO]
> minimize the previous comment not only when the plan-preview succeed but also the plan-preview result has an error (because not minimized before)

To realize this, It would be nice to gather these process into one method 👀 
- find latest comment
- makeCommentBody
- do comment
- minimizeComment

Then we only call the method regardless of failure or success. WDYT?

Reply:
@ffjlabo 
Yes! Exactly!
I will gather these process.

Reply:
@ffjlabo 
Applied!!
https://github.com/pipe-cd/pipecd/pull/4683/commits/74a9d8e231a532e2c86ab8e137119dafe86c0988

Reply:
@TakumaKurosawa Sorry for late reply 🙏 It looks good 👍